### ISSUE TITLE
Unexpected value

### ISSUE THREAD

**Issue by Firesphere** [ironsheep/lovelace-rpi-monitor-card#39]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
Latest

**Last working release (if known):**
Unknown

**Browser and Operating System:**
RPi 4
Latest Home Assistant in Docker

**Description of problem:**

All cards have the following:
![image](https://user-images.githubusercontent.com/680570/217964429-6afdce5c-bae1-4ed1-bdb1-1669addd648e.png)

**Javascript errors shown in the web inspector (if applicable):**

none

**Additional information:**
Did a git-pull on all the pis for the related MQTT daemon

**Comment by ironsheep** [comment]
Thanks for reporting this.

Yes, this latest release is a work in progress as there are underlying issues with the libraries that I'm trying to track down.

If you take a look at the release notes https://github.com/ironsheep/lovelace-rpi-monitor-card/releases/tag/v1.3.0
you will see that I'm asking for anyone seeing the "{unexpected value}..." to check the browser development tools console log report back what values your cards are getting.   I appreciate all help you can provide.

I'm continuing to chase the underlying crashes until I find a fix. You may see a number of updates to this card and to the MQTT Daemon while I get this all sorted out.

**Comment by Firesphere** [comment]
I have none of those messages in my console:
![image](https://user-images.githubusercontent.com/680570/218222272-8d81fc27-e184-4cfa-bbe8-538926aa6372.png)
Only an error which doesn't say what origin it has.

**Comment by Firesphere** [comment]
Wait, update:
```
 HCUTE (DBG) (Jewel) card_timestamp_value=[NaN-NaN-NaN] [rpi-monitor-card.js:218:3908](https://homassistant.local/hacsfiles/lovelace-rpi-monitor-card/rpi-monitor-card.js)
 ```
And:
```
Uncaught (in promise) TypeError: can't access property "data", this._$AA.nextSibling is null
    g rpi-monitor-card.js:31
    _$AI rpi-monitor-card.js:31
    p rpi-monitor-card.js:31
    $ rpi-monitor-card.js:31
    _$AI rpi-monitor-card.js:31
    k rpi-monitor-card.js:31
    _$AI rpi-monitor-card.js:31
    p rpi-monitor-card.js:31
    $ rpi-monitor-card.js:31
    _$AI rpi-monitor-card.js:31
    Do rpi-monitor-card.js:37
    update rpi-monitor-card.js:37
    performUpdate rpi-monitor-card.js:25
    scheduleUpdate rpi-monitor-card.js:25
    _$Ej rpi-monitor-card.js:25
    requestUpdate rpi-monitor-card.js:25
    set rpi-monitor-card.js:25
    value hui-masonry-view.ts:357
    value hui-masonry-view.ts:357
    performUpdate app.98f1a6e8.js:1513
    scheduleUpdate app.98f1a6e8.js:1513
    _$Ej app.98f1a6e8.js:1513
    requestUpdate app.98f1a6e8.js:1513
    set app.98f1a6e8.js:1513
    value hui-root.ts:548
    _$AE app.98f1a6e8.js:1513
    performUpdate app.98f1a6e8.js:1513
    scheduleUpdate app.98f1a6e8.js:1513
    _$Ej app.98f1a6e8.js:1513
    requestUpdate app.98f1a6e8.js:1513
    set app.98f1a6e8.js:1513
    j app.98f1a6e8.js:1513
    _$AI app.98f1a6e8.js:1513
    p app.98f1a6e8.js:1513
    $ app.98f1a6e8.js:1513
    _$AI app.98f1a6e8.js:1513
    j app.98f1a6e8.js:1513
    update app.98f1a6e8.js:1513
    performUpdate app.98f1a6e8.js:1513
    scheduleUpdate app.98f1a6e8.js:1513
    _$Ej app.98f1a6e8.js:1513
    requestUpdate app.98f1a6e8.js:1513
    set app.98f1a6e8.js:1513
    value app-drawer-layout.js:189
    value hass-loading-screen.ts:84
    performUpdate app.98f1a6e8.js:1513
    scheduleUpdate app.98f1a6e8.js:1513
    _$Ej app.98f1a6e8.js:1513
    requestUpdate app.98f1a6e8.js:1513
rpi-monitor-card.js:31:4892
```

**Comment by ironsheep** [comment]
Replaced the timer calculation routine with my own.  Let's see if this clears this up.

**Comment by ironsheep** [comment]
Fixes committed (failing code replaced with new mech.) Fixes appear in v1.3.2 and later.
